wrangler.jsonc

```diff
@@ -10,14 +10,17 @@
 		"enabled": true
 	},
 	"assets": {
-    "directory": "./dist",
-    "binding": "ASSETS"
-  },
+		"directory": "./dist",
+		"run_worker_first": [
+			"/api/*" // API routes skip static asset handling
+		]
```

Comment:
This `run_worker_first` block isn't actually necessary in that the code will still run without this – but you will pay the (minor) penalty of the static asset handler checking any `/api/...` path before falling back to the Worker – whereas this makes it more clear (IMO) what the intent is.

It also means you can add `single-page-application` handling and be fine too.

However, feel free to remove if you like!